pkg/providers/common/common.go

```diff
@@ -26,7 +26,18 @@ const (
 	publicKeyFileName  = "eks-a-id_rsa.pub"
 )
 
-func GetAuditPolicy() string {
+func GetAuditPolicy(kubeVersion v1alpha1.KubernetesVersion) string {
```

Comment:
should this return `[]byte`?

Reply:
interesting, any specific reason why? asking because string works in this format, i haven't tried it with a byte array.

Reply:
I think `[]byte` is a better type to represent yaml content. It's also mutable so you avoid reallocation whenever it needs to change.
It's also more consistent with `yaml.Marshal`